cyences_app_for_splunk/default/macros.conf

```diff
@@ -197,6 +197,14 @@ definition = dedup host \
 definition = `cs_linux` sourcetype="usersWithLoginPrivs"
 iseval = 0
 
+[cs_linux_groups]
+definition = `cs_linux` sourcetype="groups"
+iseval = 0
+
+[cs_linux_userdetails]
+definition = `cs_linux` sourcetype="userdetails"
+iseval = 0
+
 [cs_linux_sudousers]
```

Comment:
Can we remove this macro and its usage?

Reply:
why?

Reply:
I thought, we don't need sudoers data and related stuff because it's covered as part of userdetails data now.